db/migrate/20221129094342_create_value_definitions.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# Data model for storing value definitions
+class CreateValueDefinitions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :value_definitinos, id: :uuid do |t|
+      t.default_value :string
+      t.string :ref_id
+      t.uuid :benchmark_id, null: false
+      t.string :title
+      t.text :description
+      t.string :value_type
+      t.numeric :lower_bound, default: nil
+      t.numeric :upper_bound, default: nil
```

Comment:
<img width="959" alt="Screenshot 2022-11-29 at 2 29 20 PM" src="https://user-images.githubusercontent.com/87209745/204641046-a130246d-e85b-4eaa-9d6c-103665f79d0c.png">

It looks like `numeric` is similar to `decimal` or `float`. I think this should be fine because lower_bound and upper_bound can be floats/decimals.

Reply:
Thanks!